ui/admin/tests/acceptance/groups/update-test.js

```diff
@@ -0,0 +1,100 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, click, find, fillIn } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { Response } from 'miragejs';
+import {
+  authenticateSession,
+  // These are left here intentionally for future reference.
+  //currentSession,
+  //invalidateSession,
+} from 'ember-simple-auth/test-support';
+
+module('Acceptance | groups', function (hooks) {
```

Comment:
Do we need a test case here for when the user does not have the update permission?

Reply:
good point. I will add a test case for that.